website/client/components/notifications.vue

```diff
@@ -199,6 +199,9 @@ export default {
     userClassSelect () {
       return !this.user.flags.classSelected && this.user.stats.lvl >= 10;
     },
+    userHasClass () {
+      return this.user.stats.lvl >= 10 && this.user.flags.classSelected && !this.user.preferences.disableClasses;
```

Comment:
Love it! That will be useful in lots of other places. It's probably best to leave out the level check though because a . IIRC.

Reply:
Sorry, accidentally deleted a bunch of words while editing and didn't notice. :)

Love it! That will be useful in lots of other places. It's probably best to leave out the level check though because a level 10 player with a class who dies will be level 9 but still have their class.

Reply:
:) why leave out the level though?

Reply:
like in the other PR, we should probably use the hasClass getter in client/store/getters/members